idea/frontend/src/features/voucher/ui/duration-form/duration-form.tsx

```diff
@@ -0,0 +1,88 @@
+import { useApi } from '@gear-js/react-hooks';
+import { Button, Select } from '@gear-js/ui';
+import { useMemo } from 'react';
+
+import { Input } from '@/shared/ui';
+
+import { FIELD_NAME } from '../../consts';
+import { getMilliseconds, getPluralizedUnit, getTime } from '../../utils';
+import styles from './duration-form.module.scss';
+
+type Props = {
+  value: string;
+  onChange: (duration: string) => void;
+};
+
+const DURATION_UNIT = {
+  HOUR: 'hour',
+  DAY: 'day',
+  MONTH: 'month',
+} as const;
+
+const OPTIONS = [
```

Comment:
I think the minimum option should be 30 min cuz it's minimum available variant for issuing a voucher